README.md

```diff
@@ -28,11 +27,15 @@ E or R, when already that color, resets it to black.
 There are also keybindings with Shift and Ctrl, which you can view in-app.
 Other neat things you can do:
 
-- Hit the save button to save to a PDF.
-  The export button exports to a JSON file, which you can later open and edit.
-- Hit Ctrl+V to paste images from the system clipboard.
-  You can also drag images to the board.
-- Right-clicking to bring up a context menu to change the style.
+- Hit the export button to save to a PDF.
+  The save button exports to a JSON file, which you can later load back in to qboard.
+- Use your browser's paste function (usually Ctrl+V or Cmd+V on Mac) to paste images from the system clipboard.
```

Comment:
do we want a comma after `usually Ctrl+V`?

Reply:
no strong opinion